src/js/client/editprefset-prefSettingAdjuster-widget.js

```diff
@@ -0,0 +1,269 @@
+    },
+    bindings: { // Binding selectors: modelPaths
+        blankCheckbox: {
+            selector: "blankCheckbox",
+            path: "current.blank",
+            rules: gpii.devpmt.booleanBinderRules
+        },
+        valueInput: {
+            selector: "valueInput",
+            path: "current.value",
+            rules: {
+                domToModel: {
+                    "": {
+                        transform: {
+                            type: "gpii.devpmt.prefSettingAdjuster.binderToValueInput",
+                            inputPath: "",
+                            schema: "{that}"
+                        }
+                    }
+                },
+                modelToDom: {
+                    "": {
+                        transform: {
+                            type: "gpii.devpmt.prefSettingAdjuster.valueInputToBinder",
+                            inputPath: "",
+                            schema: "{that}"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    templates: {
+        initial: "editprefset-prefSettingAdjuster-widget"
+    },
+    invokers: {
+        saveUpdateValue: {
+            funcName: "gpii.devpmt.prefSettingAdjuster.saveUpdateValue",
+            args: ["{that}", "{editPrefs}"]
+        },
+        cancelEditing: {
+            funcName: "gpii.devpmt.prefSettingAdjuster.cancelEditing",
+            args: ["{that}"]
+        },
+        watchInputKeys: {
+            funcName: "gpii.devpmt.prefSettingAdjuster.watchInputKeys",
+            args: ["{that}", "{arguments}.0"]
+        },
+        updateBlankDisabling: {
+            funcName: "gpii.devpmt.prefSettingAdjuster.updateBlankDisabling",
+            args: ["{that}"]
+        }
+    },
+    markupEventBindings: {
+        okButton: {
+            method: "click",
+            args: "{that}.saveUpdateValue"
+        },
+        cancelButton: {
+            method: "click",
+            args: "{that}.cancelEditing"
+        },
+        valueInput: {
+            method: "keypress",
+            args: "{that}.watchInputKeys"
+        }
+    },
+    listeners: {
+        onMarkupRendered: [{
+            funcName: "fluid.focus",
+            args: ["{that}.dom.valueInput"]
+        },
+        {
+            func: "{that}.updateBlankDisabling"
+        },
+        {
+            funcName: "gpii.devpmt.prefSettingAdjuster.updateBooleanValueLabel",
+            args: ["{that}"]
+        }]
+    },
+    modelListeners: {
+        "current.blank": {
+            func: "{that}.updateBlankDisabling"
+        },
+        "current.value": [{
+            funcName: "gpii.devpmt.prefSettingAdjuster.updateBooleanValueLabel",
+            args: ["{that}"]
+        }]
+    }
+});
+
+gpii.devpmt.prefSettingAdjuster.updateBooleanValueLabel = function (that) {
+    // if (that.model.metadata.schema.type === "boolean") {
+    jQuery("#pmt-boolean-value").css("display","inline");
```

Comment:
Can you move the reference of "#pmt-boolean-value" into the component "selectors" option rather than hardcoding it here?

Also, be more specific with function arguments rather than passing in "{that}". This comment applies to the entire pull request.

Thanks.

Reply:
Removed the remaining selectors.